icu4c/source/test/cintltst/creststn.c

```diff
@@ -2185,7 +2185,7 @@ static void TestFallback(void)
         UResourceBundle* myResB = ures_open(NULL,"no_NO_NY",&err);
         UResourceBundle* resLocID = ures_getByKey(myResB, "Version", NULL, &err);
         const UChar* version = NULL;
-        static const UChar versionStr[] = u"45"; // 45 in nn_NO or in a parent bundle/root
+        static const UChar versionStr[] = u"46"; // 45 in nn_NO or in a parent bundle/root
```

Comment:
You probably want to update the comment here too.  Or if not, can you explain what's going on here?

Reply:
Duh! Will do!